### ISSUE TITLE
[xmldom error]  invalid doc source processing a xlsx file

### ISSUE THREAD

**Issue by ladrians** [harshankur/officeParser#39]
I need to process from a system this file which I have no control over.

[EZSHARE-2105577834-4.xlsx](https://github.com/user-attachments/files/16515964/EZSHARE-2105577834-4.xlsx)

I am getting the following error using `4.1.1` version

```
[xmldom error]  invalid doc source 
@#[line:0,col:undefined]
[OfficeParser]: TypeError: Cannot read properties of undefined (reading 'getElementsByTagName')
```

any idea what the problem may be?

**Comment by harshankur** [comment]
@ladrians 
Thank you for being so patient and I am sorry to be so late in fixing this.
It should be fixed with 4.1.2 now. Please reopen this issue if it hasn't.

**Comment by ladrians** [comment]
great, that case worked, thanks!
